server/build.gradle

```diff
@@ -122,9 +120,11 @@ tasks.withType(JavaCompile) {
     }
 }
 
-processResources {
-    into('public') {
-        from configurations.publicResources
+if(System.getenv("SKIP_WEB_UI") == null) {
```

Comment:
might be worth defining a chore to get the build a bit more developer friendly.

Reply:
I'm not calling this out as anti-developer friendly, to be clear....I just know this was a bi-product of developer annoyance and wanted to call out that we need to try to fix it. ;-)